src/js/tabs/vtx.js

```diff
@@ -12,8 +12,71 @@ TABS.vtx = {
     analyticsChanges: {},
     updating: true,
     env: new djv(),
+
+    get VTX_TYPE_SMARTAUDIO() { return 3; },
```

Comment:
Getting fancy here ;-)

Reply:
lol can't use "3" two times in different places :)